package.json

```diff
@@ -27,6 +27,7 @@
     "dev:bun": "bun --bun ./bin/nuxi.mjs dev ./playground",
     "lint": "eslint . && prettier --check src",
     "lint:fix": "eslint --fix . && prettier --write src",
+    "play:nuxi": "cd ./playground && node ../bin/nuxi.mjs",
```

Comment:
```suggestion
    "play": "./bin/nuxi.mjs --cwd ./playground",
```

Reply:
Thanks!
Played with that a bit and it seems like we need to pass commands before the --cwd flag, so e.g.
`node ./bin/nuxi.mjs analyze --cwd ./playground`, but then we loose the ability to feed the command we want at the console level.
With the original suggestion you were able to execute `pnpm play cleanup` or any other nuxi command directly.
Hope that makes sense

Reply:
Yeah thanks for checking. I think there is area to improve for this to make it possible.

Reply:
@warflash 
I had the same struggle when first playing with the cli locally. Turns out the simplest solution I could find was to run it like so:

from the playground dir:

```bash
    ../bin/nuxi.mjs <whatever normally come after running nuxi>
```

Reply:
Yes, that's what i also normally do today. I think point of this PR, (which is also valid for real projects) is to prepare a generic command that allows nuxi operating in separate directory. There are few different ways such as `cd` to the dir or directly `nuxi <dev> [dir]` (limitation is, some command only accept `--dir`)